day_6/exercises/burrito.js

```diff
@@ -16,12 +16,23 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  addTopping(topping){
+    this.toppings.push(topping);// adds a push method to and a topping
+  }
+  removeTopping(topping){
+    var indextoRemove = this.toppings.indexOf(topping)
```

Comment:
Generally, you'll want to camelCase your variables in JS. For instance, `indextoRemove` would become `indexToRemove`.